src/OpenTracing/Mock/MockScopeManager.php

```diff
@@ -16,9 +16,11 @@
     /**
      * {@inheritdoc}
      */
-    public function activate(Span $span)
+    public function activate(Span $span, $finishSpanOnClose = true)
     {
-        $this->scopes[] = new MockScope($this, $span);
+        $scope = new MockScope($this, $span, $finishSpanOnClose);
+        $this->scopes[] = $scope;
```

Comment:
not familiar with php syntax - what does this mean? looks like assigning scalar to array.

Reply:
This means to append entry to an array